### ISSUE TITLE
Will you add standalone for core Fuse-Zx Spectrum too?

### ISSUE THREAD

**Issue by Vecinagamer** [noword/Emu4VitaPlus#15]
Good afternoon everyone, the independent Fuse-Zx spectrum core would be wonderful if it is possible of course, greetings.
Thank you so much.

The core of the Fuse zx-spectrum that works well is version 1.9.0 of retroarch, it works very fine and very well

**Comment by noword** [comment]
Can you test it. If had any problem, tell me the detail.
[fuse4VitaPlus_0.10_log.vpk.zip](https://github.com/user-attachments/files/18191052/fuse4VitaPlus_0.10_log.vpk.zip)

**Comment by noword** [comment]
Plz test it. If had any problem, tell me the detail.
[fuse4VitaPlus_0.10.vpk.zip](https://github.com/user-attachments/files/18193011/fuse4VitaPlus_0.10.vpk.zip)

**Comment by Vecinagamer** [comment]
I'm going to install it on psvita and I'll tell you about the performance.
You are a phenomenon, thank you very much.
Do you have a site other than PayPal where you can donate for your work.
Number counts, bizum?

**Comment by Vecinagamer** [comment]
It does not load any Zx-Spectrum companion files, it gives an error

**Comment by Vecinagamer** [comment]
Zx-Spectrum now works great,fantastic Version 0.12 wonderful Zx-Spectrum,Wonderful,,,yeahhhhhhh @noword Fantastic yeahhhhhhhhhhhhhhhhhhhhhhhhhhhhh

**Comment by Snowbro88** [comment]
Yes I have tested it today and the games run without problems, but it seems to have the problem where when closing a game and loading another with the same core Emu4vita++ crashes.
